### PR TITLE
Correcting typo in Unsupervised Learning section

### PR DESCRIPTION
**Pull request by jroberayalas**
Correcting a (relatively important) typo.

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1368/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1368/

File: chapter_introduction/index.md [hunk 1]
```diff
@@ -980,7 +980,7 @@ All the examples so far were related to *Supervised Learning*,
 i.e., situations where we feed the model a giant dataset
 containing both the features and corresponding target values.
 You could think of the supervised learner as having
-an extremely specialized job and an extremely anal boss.
+an extremely specialized job and an extremely banal boss.
```
Comment (AnirudhDagar) [inline chapter_introduction/index.md:983]: banal: so lacking in originality as to be obvious and boring.

Comment (AnirudhDagar) [reply chapter_introduction/index.md:983]: This is actually not a typo! xD

**Comment by astonzhang** [comment]
Though we mean "anally retentive",  I think "banal" can be less confusing :)

**Pull request merged**